packages/manager/modules/dbaas-logs/src/logs/detail/streams/archives/streams-archives.html

```diff
@@ -49,6 +49,24 @@ <h6 class="oui-heading_6">
                 data-translate-values="{{$row}}"
             ></div>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title="::'streams_archives_encryption_keys' | translate"
+            data-helper="::'streams_archives_encryption_keys_helper' | translate"
+            data-property="filename"
+        >
+            <ul
+                class="p-1 m-0"
+                data-ng-if="ctrl.archivesEncryptionKeys.hasOwnProperty($row.archiveId)"
+            >
+                <li
+                    class="p-0 m-0"
+                    style="list-style-type: none;"
+                    data-ng-repeat="key in ctrl.archivesEncryptionKeys[$row.archiveId] track by $index"
+                >
+                    {{ key }}
```

Comment:
**nitpick:** Prefer using `ng-bind` over `{{ }}` as bind is better performant than interpolation.

Reply:
Fixed